blockly-core/core/ui/block.js

```diff
@@ -254,14 +254,14 @@ Blockly.Block.isFreelyDragging = function() {
 
 /**
  * Wrapper function called when a mouseUp occurs during a drag operation.
- * @type {Array.<!Array>}
+ * @type {bindData}
```

Comment:
+1 for PascalCase for JS types. Good point about making our codebase more accessible to contributors! Note that there is some headache associated with changing the case of any filenames.